pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1311,3 +1311,17 @@ def test_dropna(fill_value):
     df = pd.DataFrame({"a": [0, 1], "b": arr})
     expected_df = pd.DataFrame({"a": [1], "b": exp}, index=pd.Int64Index([1]))
     tm.assert_equal(df.dropna(), expected_df)
+
+
+def test_maxmin():
```

Comment:
this tests is not relevant to this PR. it has filtered through from your other PR